source/production/F0.Minesweeper.Components/Difficulty.razor

```diff
@@ -0,0 +1,9 @@
+<div>
+	<label for="difficulty">Difficulty</label>
```

Comment:
I guess there's one future issue to be had for string resource management? Or will we do such things in #33 ?

Reply:
yes, I tried to goof around a little bit - :)
not sure if we will do translations or any cultural thing, but on the other hand, it's more like a "maybe?!". 

:)